community/py-opencl/APKBUILD

```diff
@@ -2,31 +2,55 @@
 # Maintainer: Stuart Cardall <developer@it-offshore.co.uk>
 pkgname=py-opencl
 _pkgreal=${pkgname/-/}
-pkgver=2016.2.1
+pkgver=2017.1.1
 pkgrel=0
 pkgdesc="Python wrapper for OpenCL"
 url="https://pypi.python.org/pypi/pyopencl"
 arch="all !ppc64le !s390x"
 license="MIT"
-depends="python2"
-depends_dev="$pkgname"
-makedepends="python2-dev py-setuptools py-mako py-numpy ctags libffi-dev mesa-dev opencl-headers opencl-icd-loader-dev"
-subpackages="$pkgname-dev"
+depends="py-mako"
+makedepends="python2-dev python3-dev py-setuptools py-mako py-numpy ctags libffi-dev mesa-dev opencl-headers \
+		opencl-icd-loader-dev py-cparser py-cffi"
+checkdepends="py-six py-decorator pytest py-py"
+subpackages="py2-${pkgname#py-}:_py2 py3-${pkgname#py-}:_py3 $pkgname-dev"
 source="https://files.pythonhosted.org/packages/source/${_pkgreal:0:1}/$_pkgreal/$_pkgreal-$pkgver.tar.gz"
 builddir="$srcdir/$_pkgreal-$pkgver"
 
 build() {
 	cd "$builddir"
-	python2 configure.py --cl-enable-gl --python-exe=python2 || return 1
-	make || return 1
-	python2 setup.py test
+	python2 setup.py build
+	python3 setup.py build
 }
 
 package() {
+	mkdir -p "$pkgdir"
+	mkdir -p "$pkgdir-dev"
```

Comment:
I don't think this should be necessary? If it is, `default_dev` is doing nothing which means your `-dev` subpackage is empty anyway.